cheroot/ssl/builtin.pyi

```diff
@@ -14,7 +14,10 @@ class BuiltinSSLAdapter(Adapter):
         certificate_chain: _t.Any | None = ...,
         ciphers: _t.Any | None = ...,
         *,
-        private_key_password: str | bytes | None = ...,
+        private_key_password: _t.Callable[[], bytes | str]
```

Comment:
Add `import collections.abc as _c` at the top and use it instead of the deprecated type:
```suggestion
        private_key_password: _c.Callable[[], bytes | str]
```